src/cfnlint/core.py

```diff
@@ -113,16 +127,19 @@ def create_parser():
         action='store_true', help='list all the rules'
     )
     parser.add_argument(
-        '-r', '--regions', dest='regions', default=['us-east-1'], nargs='*',
+        '-r', '--regions', dest='regions', nargs='+', default=[],
+        type=comma_separated_arg, action='extend',
         help='list the regions to validate against.'
     )
     parser.add_argument(
-        '-a', '--append-rules', dest='append_rules', default=[], nargs='*',
+        '-a', '--append-rules', dest='append_rules', nargs='+', default=[],
+        type=comma_separated_arg, action='extend',
         help='specify one or more rules directories using '
              'one or more --append-rules arguments. '
     )
     parser.add_argument(
-        '-i', '--ignore-checks', dest='ignore_checks', default=[], nargs='*',
+        '-i', '--ignore-checks', dest='ignore_checks', nargs='+', default=[],
```

Comment:
I'm not sure this is handled today so take this with a grain of a salt.  If someone wanted to ignore all checks that could exist in a file would they be able to just add `-i` or `--ignore-checks` with no additional options?  Would we be able to catch that or would they need to add a value like `-i NONE`.

I have a patch in for Atom plugin right now as that was sending empty values in for a few of these and that needs to get fixed.